ffs/integrationtest/integration_test.go

```diff
@@ -40,9 +39,16 @@ import (
 	"github.com/textileio/powergate/wallet"
 )
 
+const (
+	tmpDir = "/tmp/powergate"
```

Comment:
This is fixed since it's important for the mapping of the docker-volume with the _"should be local"_ lotus node.